docs/docs/tutorials/cost-estimate-report.md

```diff
@@ -0,0 +1,116 @@
+---
+id: cost-estimate-report
+title: Create a cost estimate report
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## What is Infracost?
+
+[Infracost](https://www.infracost.io/) is a tool that enables developers to analyze their cloud cost estimates using their Terraform templates. Nebari wraps the `infracost` command-line interface, which is free and open-source, to provide cost estimates for your Nebari deployments across major cloud providers from your terminal.
```

Comment:
```suggestion
[Infracost](https://www.infracost.io/) is a tool that enables developers to analyze their cloud cost estimates using their Terraform templates. Nebari wraps the `infracost` command-line interface, which is free and open-source, to provide cost estimates for your Nebari deployments across major cloud providers.
```

Reply:
This sentence was just too long. I removed a bit we didn't need to get the length down a little.